.circleci/config.yml

```diff
@@ -293,6 +302,30 @@ jobs:
           to: s3://${AWS_BUCKET}/widget-demo/production
           overwrite: true
 
+  tag_for_production:
+    executor: main-executor
+    working_directory: ~/react-widgets
+    steps:
+      - checkout
+      - run: git remote add upstream git@github.com:webex/react-widgets.git
+      - run: git fetch --all --tags --prune
+      - run:
+          name: 'Grab Version Number'
+          command: |
+            echo export VERSION_NUMBER=$(curl 'https://code.s4d.io/widget-demo/production/.version') >> $BASH_ENV
```

Comment:
If the curl fails here, does this command fail gracefully?

Reply:
nope
You got a quick fix?

Reply:
The only thing I can think of is just separating the curl command out and making it error if it fails.